romac4j/src/test/java/com/rakuten/rit/roma/romac4j/AListPluginTest.java

```diff
@@ -12,7 +12,7 @@
     static AListPluginClient ac = null;
 
     static void oneTimeSetUp() throws Exception {
-        rc = new RomaClient("localhost_11211");
+        rc = new RomaClient("localhost_11311");
```

Comment:
Why change on ports are needed?

Reply:
I don't know the reason but 11211 port is unstable on travis ci.